package.json

```diff
@@ -27,6 +27,7 @@
   "homepage": "https://github.com/WATonomous/infra-config#readme",
   "description": "A website for WATcloud",
   "dependencies": {
+    "@giscus/react": "^3.0.0",
```

Comment:
Are there no changes in package-lock.json?

Reply:
Yes there was I'll add it